docs/DeveloperGuide.md

```diff
@@ -1379,13 +1379,36 @@ Deleting a student while only some students are being shown
 
 { end of `manual_testing#deleting_a_student` written by: Marcus Tan Wei }
 
-{ start of `manual_testing#adding_a_session` written by: ________ }
+{ start of `manual_testing#adding_a_session` written by: Alvin Chee Teck Weng }
 
 ### 11.4. Adding a session
 
-todo
+Adding a session while all sessions are being shown.
 
-{ end of `manual_testing#adding_a_session` written by: ________ }
+   1. Prerequisites: Multiple (but less than 100) sessions in the session list.
+ 
+   1. Test case: `addses s/Tutorial 6 d/12/7/2020`<br>
+      Expected: A session is added to the bottom of the session list. A success message including the particulars of the added student is shown in the result box.
+      
+   1. Test case: `addses s/Tutorial 7 d/25/12/2020`<br>
+      Expected: A session is added to the top of the session list. A success message including the particulars of the added student is shown in the result box.
```

Comment:
Maybe this line could be changed to "Similar to previous"?

Reply:
One should be top one should be bottom due to order of the date